pkg/config/remote/uptane/partial_client.go

```diff
@@ -167,7 +170,22 @@ func (c *PartialClient) Update(response *pbgo.ClientGetConfigsResponse) error {
 	}
 	err = c.validateAndUpdateTargets(response.Targets.Raw)
 	if err != nil {
-		return err
+		if errors.Is(err, verify.ErrInvalid) {
```

Comment:
I agree with Dario.

Typed errors are nice when the caller can take different error handling actions based on the class of error. In this case though we're just logging the error in every case so we aren't really handling it any differently.

In addition, if the caller of Update logs the error returned from update (which it likely does) now we have duplicated log information. In a sense, the `PartialClient` should be treated as "library" code and we should let the caller decide how it wants to log error information. We might want to appropriately wrap the error so that there's appropriate context, but I don't think we should log here as well.

Reply:
Done in ebb22c5

@ameske : logging? `fmt.Errorf` does not log, it simply wraps the error.